### PR TITLE
Update companies.md

### PR DESCRIPTION
**Pull request by liashahnazaryan**
# Description
I've added several new MT companies to the company section. 
There's also [Ciceron](https://www.crunchbase.com/organization/ciceron-inc) from China, but, as I understand, their website has been under construction since they hired new CEO, so I'll wait to get updates on their status, and if they're still active, I'll add them to the list as well. 

Also, several companies have their own MT products. So, I was thinking about adding them to the 'Products' section. One of them is [LexMachina](https://lex-machina.ch/), created by [Hieronymus](https://hieronymus.ch/en/), and the other one is [Letr](https://www.letr.ai/), created by [Twigfarm](https://en.twigfarm.net/). So let's discuss if we need them listed in the 'Products' section or not. What do you think, @bittlingmayer and @cefoo?

## Type of PR
Adding new company profiles to the list

### CODE REVIEW

**Comment by cefoo** [comment]
Hi @liashahnazaryan!

Yes, I agree with adding those companies to the products.md.

The key info we need about those products is:
- Do they offer MT services or an MT-related service? Is it an MT aggregator? (I think we should add MT aggregators to that page, such as ModelFront, LDX Hub, or Eden AI, and thus have two lists: MT engines and MT aggregators)
- If they offer MT services, is it through an API? Is the API self-serve?

**Comment by bittlingmayer** [comment]
Another one: https://vidby.com/

**Comment by bittlingmayer** [comment]
Certainly Products needs to be its own page.

So e.g. Google Translate (and the Google Cloud Translate API is a separate product).

File: industry/companies.md [hunk 1]
```diff
@@ -35,6 +41,12 @@ Raised pre-seed from Blue Ridge Foundation and $4M from General Catalyst, AIX Ve
 - [imagetranslate.com](https://www.imagetranslate.com/)
 - [Crunchbase - ImageTranslate](https://www.crunchbase.com/organization/imgtranslate)
 
+### Zhongke Fanyu Technology
+- [zkfy.com](http://www.zkfy.com/)
+- [Crunchbase - Zhongke Fanyu Technology](https://www.crunchbase.com/organization/zhongke-fanyu-technology)
+
+Incubated by the Chinese Academy of Sciences’ Institute of Automation
```
Comment (bittlingmayer) [inline industry/companies.md:48]: If I had to bet, I'd bet this sentence will be machine-translated poorly. :-)

I'd make it:

> Incubated by the Institute of Automation at the Chinese Academy of Sciences

Or maybe just
> Supported by the Institute of Automation at the Chinese Academy of Sciences

**Review by bittlingmayer — APPROVED** [review]

**Comment by bittlingmayer** [comment]
If we have a Products page, we can be stricter about Companies.

**Comment by liashahnazaryan** [comment]
> Hi @liashahnazaryan!
> 
> Yes, I agree with adding those companies to the products.md.
> 
> The key info we need about those products is:
> 
> * Do they offer MT services or an MT-related service? Is it an MT aggregator? (I think we should add MT aggregators to that page, such as ModelFront, LDX Hub, or Eden AI, and thus have two lists: MT engines and MT aggregators)
> * If they offer MT services, is it through an API? Is the API self-serve?

Good point. I'll think about this and create a structure so that we can discuss and see what works best.

**Comment by liashahnazaryan** [comment]
> Overall, looks good to me, assuming you did the research and they meet the bar
> 
> Adding 1-line description to each company in the list is a separate (and fairly time-consuming) task.

You mean adding a line about what the companies do in terms of product, right, @bittlingmayer?

**Comment by bittlingmayer** [comment]
> You mean adding a line about what the companies do in terms of product, right, @bittlingmayer?

Right, e.g. if it's more of an MT provider, or an LSP or something else, and anything else that makes it special.

What's the one thing I need to know about this company :-)

**Pull request merged**